tests/migration_test.go

```diff
@@ -932,23 +932,49 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 		Context("with an Fedora shared NFS PVC, cloud init and service account", func() {
 			var pvName string
 			var vmi *v1.VirtualMachineInstance
-			BeforeEach(func() {
+
+			initNFS := func(isIpv4 bool) {
 				tests.SkipNFSTestIfRunnigOnKindInfra()
 				pvName = "test-nfs" + rand.String(48)
 				// Prepare a NFS backed PV
 				By("Starting an NFS POD")
 				os := string(cd.ContainerDiskFedora)
-				nfsIP := tests.CreateNFSTargetPOD(os)
+				nfsIPs := tests.CreateNFSTargetPOD(os)
 				// create a new PV and PVC (PVs can't be reused)
 				By("create a new NFS PV and PVC")
+				var nfsIP string
+				if (isIpv4) {
+					nfsIP = nfsIPs[0].IP
```

Comment:
we shouldnt assume that ipv4 is the first, in the future it can be the non primary ip, hence it wont be on place 0 / PodIP

we should call it test primary / test secondary,
this is also why it adds wear and tear that isnt needed when using DescribeTable IMHO.

Reply:
That's fine. We are not supporting single stack ipv6.
The lane was removed - https://github.com/kubevirt/project-infra/pull/642

Reply:
the comment was mainly about the previous logic that assumed IPv4 is on place 0 and was fixed by the latest commits.

now that you fixed it, the last paragraph is outdated as it said in the last line, but still i think that primary and secondary approach would remove the need of the new functions.